docs/versioned_docs/version-v3.2.0/02-Features/endpoint-validation.md

```diff
@@ -0,0 +1,215 @@
+﻿---
+title: "Endpoint validation"
+layout: default
+---
+
+# Endpoint validation
+
+We provide support for endpoint validation, when implementing your own custom web hook.
+
+- [Endpoint validation](#endpoint-validation)
+  - [Installation](#installation)
+  - [Azure Event Grid authorization](#azure-event-grid-authorization)
+    - [Enforce authorization globally](#enforce-authorization-globally)
+      - [Configuration](#configuration)
+    - [Enforce authorization per controller or operation](#enforce-authorization-per-controller-or-operation)
+      - [Configuration](#configuration-1)
+  - [Azure Event Grid subscription validation](#azure-event-grid-subscription-validation)
+    - [Enforce subscription validation per controller or operation](#enforce-subscription-validation-per-controller-or-operation)
+    - [Use subscription validation in Azure Functions](#use-subscription-validation-in-azure-functions)
+
+## Installation
+
+The features described here require the following package:
+
+> **⚠ This package used to be called `Arcus.EventGrid.Security`. Please make sure that you migrate towards this package as it's not being actively maintained beyond v3.2.**
+
+```shell
+PM> Install-Package Arcus.EventGrid.Security.WebApi
+```
+## Azure Event Grid authorization
+
+Azure Event Grid authorization allows to secure your webhook with a secret key (taken from the query string or an HTTP header). 
+This is needed, because Azure Event Grid is send a validation request to a newly configured web hook, in order to prevent people leveraging Azure Event Grid to bring down a 3rd party API. 
```

Comment:
```suggestion
This is required, because Azure Event Grid sends a validation request to a newly configured web hook in order to prevent people leveraging Azure Event Grid to bring down a 3rd party API. 
```

Reply:
Done in https://github.com/arcus-azure/arcus.eventgrid/pull/232/commits/ce25ecdfee823a37662b07ef1fded7115a06a213